src/wwElement.vue

```diff
@@ -158,6 +160,189 @@
+                    :class="{ 'is-highlighted': richEditor.isActive('table') }"
+                    @click="insertColumn('before')"
+                    :disabled="!isEditable"
+                    v-if="menu.table && richEditor.isActive('table')"
+                >
+                    <svg class="icon" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path
+                            fill-rule="evenodd"
+                            clip-rule="evenodd"
+                            d="M14 3H24V27H17.9297C17.4943 27.7528 16.9396 28.4281 16.2916 29H24C25.1046 29 26 28.1046 26 27V3C26 1.89543 25.1046 1 24 1H14C12.8954 1 12 1.89543 12 3V15.0619C12.6986 15.149 13.3693 15.3262 14 15.5815L14 3Z"
+                            fill="currentColor"
+                        />
+                        <circle cx="11" cy="23" r="7" fill="currentColor" />
+                        <line
+                            y1="-1"
+                            x2="7"
+                            y2="-1"
+                            transform="matrix(-1 0 0 1 13 8)"
+                            stroke="currentColor"
+                            stroke-width="2"
+                        />
+                        <path d="M4 7L8.5 11.3301L8.5 2.66987L4 7Z" fill="currentColor" />
+                        <line x1="7" y1="23" x2="15" y2="23" stroke="white" stroke-width="2" />
+                        <line x1="11" y1="27" x2="11" y2="19" stroke="white" stroke-width="2" />
+                    </svg>
+                </button>
+                <button
+                    type="button"
+                    class="ww-rich-text__menu-item"
+                    :class="{ 'is-highlighted': richEditor.isActive('table') }"
+                    @click="insertColumn('after')"
+                    :disabled="!isEditable"
+                    v-if="menu.table && richEditor.isActive('table')"
+                >
+                    <svg class="icon" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path
+                            fill-rule="evenodd"
+                            clip-rule="evenodd"
+                            d="M6 3H16V15.5815C16.6307 15.3262 17.3014 15.149 18 15.0619V3C18 1.89543 17.1046 1 16 1H6C4.89543 1 4 1.89543 4 3V27C4 28.1046 4.89543 29 6 29H13.7084C13.0604 28.4281 12.5057 27.7528 12.0703 27H6L6 3Z"
+                            fill="currentColor"
+                        />
+                        <circle cx="19" cy="23" r="7" fill="currentColor" />
+                        <line x1="17" y1="7" x2="24" y2="7" stroke="currentColor" stroke-width="2" />
+                        <path d="M26 7L21.5 11.3301L21.5 2.66987L26 7Z" fill="currentColor" />
+                        <line x1="15" y1="23" x2="23" y2="23" stroke="white" stroke-width="2" />
+                        <line x1="19" y1="27" x2="19" y2="19" stroke="white" stroke-width="2" />
+                    </svg>

```

Comment:
Peut etre faire un sous element Icon qui prend en prop le nom de l'icon, et dans l'element tu résous le bon svg en fonction du nom (avec des v-if), pour que le template du rich text reste un peu digeste

Reply:
yes I'll update the PR with this update, I did that to avoid rendering several vue components

Reply:
@AlexisJC Done, added a TableIcon vue component with icon props